core/src/main/scala/com/evolutiongaming/kafka/flow/PartitionFlowConfig.scala

```diff
@@ -66,4 +67,27 @@ object PartitionFlowConfig {
       */
     case object Sequential extends RecoveryMode
   }
+
+  /** Controls how timers are executed. Timers are executed in parallel, but the number of concurrent executions can be
+    * limited by the specified parameter.
+    */
+  sealed trait TimersExecutionMode
+
+  object TimersExecutionMode {
+
+    /** Execute timers in parallel without a limit on the number of concurrent executions. For each timer (corresponds
+      * to a key) a fiber will be spawned. This is the fastest mode, but it might consume excessive CPU resources
+      * depending on a number of keys currently held in memory.
+      */
+    case object Unbounded extends TimersExecutionMode
+
+    /** Execute timers in parallel with a limit on the number of concurrent executions. For each timer (corresponds to a
+      * key) a fiber will be spawned, but a total number of concurrent fibers will not exceed the specified limit. This
+      * mode is slower than `Unbounded`, but it allows fine-tuning the number of concurrent fibers to prevent CPU
+      * starvation and overwhelming the underlying storage with too many parallel executions.
+      * @param parallelism
+      *   the upper bound on the number of concurrent fibers
+      */
+    case class Bounded(parallelism: Int) extends TimersExecutionMode
```

Comment:
why don't we just bound to number of CPU * multiplier? hence not operate with a limit but rather a multiplier?

Reply:
Then it would depend on external conditions, such as the configuration of an application in k8s cluster. With the absolute number, it's quite easy to calculate an absolute max number of spawned fibers per second, e.g. in case Cassandra used as a snapshot storage; all we need to know is 1) this limit (which is applied per partition); 2) a rough estimate of Cassandra query latency; 3) the number of partitions. And that is what we need in most of our use cases of this framework.  

Someone changing `requests`/`limits` of the application externally won't affect this, and an application will still produce consistent load on Cassandra whereas having a multiplier makes one change it any time one increases or decreases the number of vCPUs. This is especially relevant for increasing a number of vCPUs as it might hurt the storage proportionally.